### ISSUE TITLE
Lower-left corner links are indistinguishable

### ISSUE THREAD

**Issue by lee0c** [hometown-fork/hometown#1379]
### Steps to reproduce the problem

1. Using tab navigation and a screen reader, tab to the lower-left corner links (Web UI, full screen)
2. Notice while tabbing through that there are three links read out as 'About' (server about, hometown about, and masto about) and two links read out as 'View source code' (hometown source code vs masto source code)

### Expected behaviour

Links should be differentiated for screen reader users

### Actual behaviour

Links are indistinguishable without accompanying visual text

### Detailed description

These can be disambiguated using `aria-label`

### Mastodon instance

flipping.rocks

### Mastodon version

v4.2.17+hometown-1.1.2

### Browser name and version

Firefox 145.0.2

### Operating system

Win11

**Comment by lee0c** [comment]
This is something I can help with.

**Comment by lee0c** [comment]
Fixed in mainline masto!

**Comment by nachtjasmin** [comment]
Still present, tested with `v4.5.11+hometown-1.2.1`.
